Jenkinsfile.ci

```diff
@@ -234,6 +234,7 @@ def linuxTesting(file, env_name, network_name) {
 
         sh "cp libindy/target/debug/libindy.so wrappers/java/lib"
```

Comment:
You also should update docker files ubuntu and amazon placed here libindy/ci. 
After that, on your local machine, you can check your steps inside docker.
Not it doesn't work for me